docs/entities.rst

```diff
@@ -596,3 +596,169 @@ The endpoint is used to delete a single Entity or multiple Entities. The IDs of
 **Response**
 
 Status: ``204 No Content``
+
+Import Entities from CSV
+-------------------------
+
+.. raw:: html
+
+	   <pre class="prettyprint"><b>POST</b> /api/v2/entity-lists/&lt;entity_list_id&gt;/import-entities</pre>
```

Comment:
Since we have entities under the path `/entity-list/<pk>/entities` should we allow import on the same endpoint? If not, perhaps consider `/entity-list/<pk>/entities-import` instead.

Reply:
I would expect that it will pick properties from the CSV file being uploaded. Is there an extra step that was skipped API-wise? Is that documented?

Reply:
@ukanga Yes properties are picked from the CSV uploaded. Which CSV did you use? The properties must exist prior

Reply:
@ukanga You might have missed to create the properties by uploading a registration form

Reply:
This [PR adds a properties endpoint](https://github.com/onaio/onadata/pull/2945)